src/main/java/net/mcreator/ui/gradle/GradleErrorDialogs.java

```diff
@@ -63,24 +64,22 @@ else if (errorCode == GradleErrorCodes.JAVA_INVALID_VERSION)
 
 		return errorCode;
 	}
-	
+
 	private static final String MESSAGE_TITLE = "Gradle task failed with error";
```

Comment:
I meant inline as instead of doing L10N here, do it where MESSAGE_TITLE and remove MESSAGE_TITLE field.

Reply:
Oh ok 👍